lnst/Recipes/ENRT/BaseSRIOVNetnsTcRecipe.py

```diff
@@ -106,36 +131,36 @@ def test_wide_configuration(self):
         host1, host2 = self.matched.host1, self.matched.host2
         config = super().test_wide_configuration()
 
+        # TODO: ideally we would store the sriov_devices in EnrtConfiguration
+        # instance, but the devices are used also in overriden perf_test()
+        # method that does not have reference to EnrtConfiguration.
+        # Instead we will store as class instance attribute
```

Comment:
this comment isn't accurate btw... `perf_test` receives a `lnst.RecipeCommon.Perf.Recipe.RecipeConf` which has an attribute of `parent_recipe_config`.

The `RecipeConf` object is created here: https://github.com/LNST-project/lnst/blob/master/lnst/Recipes/ENRT/BaseEnrtRecipe.py#L449 and contains a copy of the **CURRENT** `EnrtConfiguration`

the solution to store the instance attribute of `sriov_devices` is still ok IMO, just a note that the comment itself isn't accurate.

Reply:
Thanks, I'll remove the comment. I was wondering if this is hidden somewhere.

Reply:
Removed.